cognitedata/reveal
# feat: add transparency when icon is rendered behind geometry

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-666

## Description :pencil:


Icons are now rendered with 50% opacity when behind geometry.

## Test instructions :information_source:



Load some model and some 360 image colletion and see that opacity is reduced when seeing icons behind geometry.


## Checklist :ballot_box_with_check:
